CHANGELOG.md

```diff
@@ -1,7 +1,30 @@
 # Changelog
-
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+## [0.6.0] - 2022-05-28
```

Comment:
This should go under the `Unreleased` section.
The new version section should be added with a separate PR during the release phase as per internal release guidelines